File: Ambulance.BLL/Models/ActionLogModel.cs
namespace AmbulanceSystem.BLL.Models;

public class ActionLogModel
{
    public int ActionLogId { get; set; }
    public int PersonId { get; set; }
    public string? FirstName { get; set; }
    public string? SecondName { get; set; }
    public string? Action { get; set; }
}

File: public/Models/ActionLogModel.js
export class ActionLogModel {
    constructor({ actionLogId = 0, personId = 0, firstName = null, secondName = null, action = null } = {}) {
        this.actionLogId = actionLogId;
        this.personId = personId;
        this.firstName = firstName;
        this.secondName = secondName;
        this.action = action;
    }
}